packages/block-library/src/comment-template/edit.js

```diff
@@ -125,6 +130,18 @@ export default function CommentTemplateEdit( {
 	} );
 	const { rawComments, blocks } = useSelect(
 		( select ) => {
+			// Show empty comments if we don't have a postId context.
+			// The structure of the empty object as a rawComment allows
+			// to inner blocks to render the default placeholders.
+			if ( ! postId ) {
+				// We set a limit in order not to overload the editor of empty comments.
+				const defaultCommentsToShow = perPage <= 3 ? perPage : 1;
+				return {
+					rawComments: Array( defaultCommentsToShow ).fill( {
+						id: null,
+					} ),
```

Comment:
I was quite sure that this would fail if there `defaultCommentsToShow` were larger than 1, but I was wrong. 

I was concerned that [`convertToTree`](https://github.com/WordPress/gutenberg/blob/1f92999896beb98a572f46722e35b31b1de8d547/packages/block-library/src/comment-template/util.js#L30) would fail in that case, but it handles it just fine 👍

Reply:
Let's see how will work with nested default comments (as it searches for a parent id) 😅